src/internal.c

```diff
@@ -7592,6 +7660,30 @@ static INLINE void CopyNameList(byte* buf, word32* idx,
     *idx = begin;
 }
 
+
+static word32 BuildNameList(char* buf, word32 bufSz,
+        const byte* src, word32 srcSz)
+{
+    const char* name;
+    word32 nameSz, idx;
+
+    (void)bufSz;
```

Comment:
Why take in bufSz if we aren't going to use it?

Reply:
I forgot to add the length checking. I'm going to use it.

Reply:
Check